### DISCUSSION TITLE
Selecting product attributes on the FO product page should not push the current url to browser history

### DISCUSSION THREAD

**Discussion by yo7hli** [PrestaShop/PrestaShop#33313] [Ideas]
#### Is your feature request related to a problem?

With each selection of a product attribute, a new entry is added in browser history, by pushing the state instead of replacing it.
In effect, the user wanting to go back to the previous page by clicking 'Back' in the browser needs to repeatedly do that to cycle through all the history entries created.

This behaviour was introduced in this [commit](https://github.com/PrestaShop/PrestaShop/commit/991cbddd60eebfe87be2f179747cf0eba25224a0#diff-05a5bcbbe047fc337f7db38b0130621497a3a215ce3eed1586c72a549e865959R412)

#### Describe the solution you'd like

I'm not sure if this was the intended behaviour or just a regression but i think the previous behaviour was more intuitive and user friendly.

#### Alternatives you've considered

there's no alternative other than modifing core js files

#### Additional context

no context

**Comment by hibatallahAouadni** [comment]
Hello @yo7hli 

Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon. If this is important to you, we strongly suggest you consider [hiring a specialist](https://www.prestashop.com/en/experts) to help you create it. 

If you are willing to implement this feature into PrestaShop as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/), the teams will be happy to help you by ensuring that it is appropriate and coherent with the rest of the project. If that is the case, please let us know! 

Thank you

**Comment by ivaldes2786** [comment]
I have the same problem in PS 1.7.8.2, any solution?

**Comment by NeOMakinG** [comment]
Unfortunately, it's intended behavior, see https://github.com/PrestaShop/PrestaShop/issues/11281

Ping @marionf or @MatShir, what do we want?

**Comment by kpodemski** [comment]
Maybe we should make this configurable. To be honest, I don't see a "better" version in these two. It makes sense to go back to the other variant of the product. On the other hand, "Back" in the browser is used to go back to the previous page, page, not a different state of the current page 🤔

**Comment by alisamie97** [comment]
> It makes sense to go back to the other variant of the product

actually it does not, because if you want to change product attribute, you just change it by clicking on another option, back button is not for that

**Comment by MatShir** [comment]
I'm like @kpodemski. Both of them can be annoying. If we revert, someone will complain as well. Best way it is to make configurable
